topostats/default_config.yaml

```diff
@@ -0,0 +1,40 @@
+base_dir: ./
+output_dir: ./output
+warnings: ignore
+cores: 4
+quiet: false
+file_ext: .spm
+filter:
+  run: true # Options : true, false
+  channel: Height
+  amplify_level: 1.0
+  threshold_method: std_dev # Options : otsu, std_dev, absolute
+  otsu_threshold_multiplier: 1.0
+  threshold_std_dev: 1.0
+  threshold_absolute_lower: -1.0
+  threshold_absolute_upper: 1.0
+grains:
+  run: true # Options : true, false
+  gaussian_size: 2.0
+  gaussian_mode: nearest
+  absolute_smallest_grain_size: 100
+  threshold_method: std_dev # Options : otsu, std_dev, absolute
+  otsu_threshold_multiplier: 1.0
+  threshold_std_dev: 1.0
+  threshold_absolute_lower: -1.0
+  threshold_absolute_upper: 1.0
+  direction: upper # Options: upper, lower, both (defines whether to look for grains above or below thresholds or both)
+  background: 0.0
+grainstats:
+  run: true # Options : true, false
+  zrange: [0, 3]
```

Comment:
Might want to check the config file just merged as the zrange has moved along with some other bits